jobs/JOBSPROC_DUMP_MONITOR

```diff
@@ -129,10 +129,6 @@ export USHobsproc=${USHobsproc:-$HOMEobsproc/ush}
 # Paths for dump and bufr_dumplist items
 # ---------------------------------------
```

Comment:
Add a comment that indicates default vars are pre-defined with the loading of the bufr_dump module.